src/modules/helpers/index.ts

```diff
@@ -622,6 +622,29 @@ export class HelpersModule {
     return arrayCopy.slice(min);
   }
 
+  /**
+   * Returns a random value from an Enum object.
+   *
+   * @param enumObject
+   *
+   * @example
+   * faker.datatype.enum(enum Color { Red, Green, Blue}) // 'Green'
+   * faker.datatype.enum(enum Direction { North = 'North', South = 'South'}) // 'South'
+   * faker.datatype.enum(enum HttpStatus { Ok = 200, Created = 201, BadRequest = 400, Unauthorized = 401 }) // 200
```

Comment:
```suggestion
   * faker.helpers.enumValue(enum Color { Red, Green, Blue}) // 'Green'
   * faker.helpers.enumValue(enum Direction { North = 'North', South = 'South'}) // 'South'
   * faker.helpers.enumValue(enum HttpStatus { Ok = 200, Created = 201, BadRequest = 400, Unauthorized = 401 }) // 200
```

+ please make sure these actually compile